devops/aws-codebuild/Jenkinsfile.cd

```diff
@@ -177,48 +230,69 @@ pipelineWrapper({
         stage("Build artifacts") {
             builds.failFast = false
             utils.parallel builds
-        }
-
-        stage('Release artifacts') {
-            String repoName = env.SOVRIN_CORE_REPO_NAME
-            String repoComponent = env.BRANCH_NAME
-            String debsDir = 'debs'
-
-            if (isRC) {
-                repoComponent = 'rc'
-            }
-            logger.info("Debian repo component is set to: '$repoComponent'")
 
             dir(debsDir) {
                 pkgsList.each {
                     unstash name: it
                 }
             }
-            logger.info("Uploading debian packages to '${repoName}/${repoComponent}' sovrin repo: $pkgsList")
+            sh "ls -la $debsDir"
+        }
 
-            dir("sovrin-packaging") {
-                downloadPackagingUtils()
+        stage("Get packaging utils") {
+            dir(pkgUtilsDir) {
+                git branch: sovrinPackagingBranch, credentialsId: gitHubUserCredId, url: sovrinPackagingRepo
             }
+        }
 
-            withCredentials([file(credentialsId: 'SovrinRepoSSHKey', variable: 'sovrin_key')]) {
-                sh "./sovrin-packaging/upload_debs.py $debsDir $repoName $repoComponent --host $env.SOVRIN_REPO_HOST --ssh-key $sovrin_key --force-backup"
+        if (isRC) {
+            repoComponent = 'rc'
```

Comment:
At first it felt like it should be `if (!isRelease && isRC)`, however after re-reading documentation update I think it was really intended here (even if we want to release a stable version we still want to push to rc channel first). I'm leaving this comment so that you can double check if code and my understanding is correct.

Reply:
Yes. Your understanding is correct.